src/Button/DangerButton.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { darken } from 'polished';
+import { BaseTheme } from '../Theme';
+import { StyledButton } from './StyledButton';
+import { ButtonProps } from './ButtonProps';
+
+const StyledPrimaryButton = styled(StyledButton)`
+  background: ${({ theme }): string => (theme.color.background.negative.dark)};
+  color: ${({ theme }): string => (theme.color.text.light)};
+  &:hover {
+    background: ${({ theme }): string => (darken(0.2, theme.color.background.negative.dark))};
```

Comment:
as noted in seas-computing/course-planner#125, these should use the `.medium` and `.dark` values for the base and hover backgrounds, respectively

Reply:
Fixed through 224e5ef